docker/py3-native/README.md

```diff
@@ -3,7 +3,75 @@
 This README purpose is to clarify the following:
 * Which docker images the native image supports
 * The purpose of the **verifiers** folder
-* OS dependencies for each docker image that the native image supports.
+* OS dependencies for each docker image that the native image supports
+* What are native images and how to handle them
+
+
+## How To Handle Native Images
+### What is a native image?
+* Native image is a combination of a lot of docker images that scripts and integrations use. 
+* The native image contains all the OS / python dependancies that are used in the supported docker images
+* For example, given the docker image taxii2, tesseract and chromium, the native image will contain all of their OS / python dependencies. 
+* That means that every integration/script that uses taxii2, tesseract or chromium docker image can also be used in the native image and there is compatibility between them.
+
+### Required: What should I do when changing a docker-image that is supported in the native image?
+1) Check if the docker image that was changed is supported also in the native image. That can be checked in the [docker native image configuration file](https://github.com/demisto/content/blob/master/Tests/docker_native_image_config.json). **Note:** if you have changed a docker-image that is supported in the native image, the **native docker validator workflow** should fail and alert you. 
+2) If the docker image is supported by the native image, make any necessary changes to the native image to ensure it can still run the integration/script, for more info refer to [Optional: How to adjust changes in the docker image to the native image?](#optional-how-to-adjust-changes-in-the-docker-image-to-the-native-image).
```

Comment:
```suggestion
2) If the docker image is supported by the native image, make any necessary changes to the native image to ensure it can still run the integration/script. For more info refer to [Optional: How to adjust changes in the docker image to the native image?](#optional-how-to-adjust-changes-in-the-docker-image-to-the-native-image).
```